spaceship/lib/spaceship/tunes/build_train.rb

```diff
@@ -141,7 +141,28 @@ def update_testing_status!(new_value, testing_type, build = nil)
           end
         end
 
-        result = client.update_build_trains!(application.apple_id, testing_type, data)
+        begin
+          result = client.update_build_trains!(application.apple_id, testing_type, data)
+        rescue Spaceship::TunesClient::ITunesConnectError => ex
+          if ex.to_s.include?("You must provide an answer for this question")
```

Comment:
Is it possible that this text matches other "questions" than the encryption question? Is there any text more specific to the `ITSAppUsesNonExemptEncryption` key we can match on instead?